src/dotnet/commands/dotnet-test/TestRunners/TestRunner.cs

```diff
@@ -51,9 +51,10 @@ private ICommand CreateTestRunnerCommand()
             var commandArgs = _argumentsBuilder.BuildArguments();
 
             return _commandFactory.Create(
-                _testRunner,
+                $"dotnet-{_testRunner}",
                 commandArgs,
-                new NuGetFramework("DNXCore", Version.Parse("5.0")));
+                new NuGetFramework("DNXCore", Version.Parse("5.0")),
+                Constants.DefaultConfiguration);
```

Comment:
Makes total sense. :smile: 

I looked back at the code and realized that this is only for the design time piece. I misunderstood the intent of the code here.

:+1: